common/app-starters-security-common/pom.xml

```diff
@@ -29,17 +29,22 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
+			<artifactId>spring-boot-starter-actuator</artifactId>
+			<optional>true</optional>
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-actuator</artifactId>
-			<scope>test</scope>
+			<artifactId>spring-boot-starter-web</artifactId>
```

Comment:
Need to understand if we need this dependency at all...
Maybe as a `test` scope?

Reply:
looks like not needed. => removed